theano/sandbox/gpuarray/opt.py

```diff
@@ -811,7 +811,7 @@ def local_lift_abstractconv2d(node, context_name):
                                    context_name=context_name)
     inps[1] = as_gpuarray_variable(node.inputs[1],
                                    context_name=context_name)
-    return [node.op(*inps)]
```

Comment:
what was the consequence of this? I suppose an optimization warning and the op not being lifted? Is that detected by a test?